network_diffusion/tpn/tpnetwork.py

```diff
@@ -0,0 +1,62 @@
+"""A script where a temporal network is defined."""
+
+from typing import Any, Dict, List, Optional  # , Union
```

Comment:
As discussed: let's for now get rid of commented code since it's not needed

Reply:
Yes. So do I directly remove them in the next commit?